core/services/run_executor.go

```diff
@@ -97,7 +98,7 @@ func (re *runExecutor) Execute(runID *models.ID) error {
 
 		validated = true
 
-		if err := re.store.ORM.SaveJobRun(&run); errors.Cause(err) == orm.ErrOptimisticUpdateConflict {
+		if err := re.store.ORM.SaveJobRun(&run); err == orm.ErrOptimisticUpdateConflict {
```

Comment:
Is this safe?

Reply:
yeah see https://github.com/smartcontractkit/chainlink/pull/3943#discussion_r573778489 for the reasoning